modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java

```diff
@@ -278,8 +312,23 @@ public void testSessionHijacking() throws Exception {
         HttpResponse response = client.execute(getRequest);
         String resultPage2 = DataExtractUtil.getContentData(response);
         EntityUtils.consume(response.getEntity());
-        Assert.assertTrue(resultPage2.contains("Authentication Error!"), "Session hijacking is possible.");
+        Assert.assertTrue(resultPage2.contains("Authentication Error!"), "Session hijacking is possible for " +
+                "tenant domain: " + tenantDomain);
+    }
 
+    @Test(alwaysRun = true, description = "Test logout request",
+            dependsOnMethods = {"testPassiveSAML2Assertion", "testSessionHijacking"})
+    public void testSendLogoutRequest() throws Exception {
+
+        String passiveParams = "?wa=wsignout1.0&wreply=" + PASSIVE_STS_SAMPLE_APP_URL + "&wtrealm=PassiveSTSSampleApp";
+        passiveParams = getPassiveParamsForTenant(passiveParams);
+        HttpGet request = new HttpGet(this.passiveStsURL + passiveParams);
+        HttpResponse response = client.execute(request);
+
+        Assert.assertNotNull(response, "PassiveSTSSampleApp logout response is null for tenant domain: " +
+                tenantDomain);
+        int responseCode = response.getStatusLine().getStatusCode();
+        Assert.assertEquals(responseCode, 200, "Invalid Response for tenant domain: " + tenantDomain);
```

Comment:
Is asserting the status enough to gurantee a successful logout?

Reply:
After a successful logout, an empty response will be redirected to the client's reply URL. Then an authentication request will be sent by the client. So we can't test any content. So have added the logic to verify the existence of sessionDataKey in the response